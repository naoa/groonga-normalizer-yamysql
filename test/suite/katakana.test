register normalizers/yamysql

normalize NormalizerYaMySQLUnicodeCI "ァアィイゥウエォオカガキギクグケゲコゴサザシジスズセゼソゾタダチヂッツヅテデトドナニヌネノハバパヒビピフブプヘベペホボポマミムメモャヤュユョヨラリルレロヮワヰヱヲンヴヵヶｧｱｨｲｩｳｴｫｵｶｶﾞｷｷﾞｸｸﾞｹｹﾞｺｺﾞｻｻﾞｼｼﾞｽｽﾞｾｾﾞｿｿﾞﾀﾀﾞﾁﾁﾞｯﾂﾂﾞﾃﾃﾞﾄﾄﾞﾅﾆﾇﾈﾉﾊﾊﾞﾊﾟﾋﾋﾞﾋﾟﾌﾌﾞﾌﾟﾍﾍﾞﾍﾟﾎﾎﾞﾎﾟﾏﾐﾑﾒﾓｬﾔｭﾕｮﾖﾗﾘﾙﾚﾛﾜｦﾝ"

