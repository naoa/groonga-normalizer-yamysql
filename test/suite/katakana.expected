register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLUnicodeCI "ァアィイゥウエォオカガキギクグケゲコゴサザシジスズセゼソゾタダチヂッツヅテデトドナニヌネノハバパヒビピフブプヘベペホボポマミムメモャヤュユョヨラリルレロヮワヰヱヲンヴヵヶｧｱｨｲｩｳｴｫｵｶｶﾞｷｷﾞｸｸﾞｹｹﾞｺｺﾞｻｻﾞｼｼﾞｽｽﾞｾｾﾞｿｿﾞﾀﾀﾞﾁﾁﾞｯﾂﾂﾞﾃﾃﾞﾄﾄﾞﾅﾆﾇﾈﾉﾊﾊﾞﾊﾟﾋﾋﾞﾋﾟﾌﾌﾞﾌﾟﾍﾍﾞﾍﾟﾎﾎﾞﾎﾟﾏﾐﾑﾒﾓｬﾔｭﾕｮﾖﾗﾘﾙﾚﾛﾜｦﾝ"
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "アアイイウウエオオカガキギクグケゲコゴサザシジスズセゼソゾタダチジツツズテデトドナニヌネノハバパヒビピフブプヘベペホボポマミムメモヤヤユユヨヨラリルレロワワイエヲンブカケアアイイウウエオオカガキギクグケゲコゴサザシジスズセゼソゾタダチジツツズテデトドナニヌネノハバパヒビピフブプヘベペホボポマミムメモヤヤユユヨヨラリルレロワヲン",
    "types": [

    ],
    "checks": [

    ]
  }
]
